src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -18,6 +18,10 @@
 <partial asp-hide="@Model.HasVacancies" name="_NoVacanciesContent" />
 
 <div asp-show="Model.HasVacancies">
+    @if (Model.ShowBlockedProviderAlert)
+    {
+        <partial name="_BlockedProviderAlert" for="BlockedProviderAlert" />
```

Comment:
I initially tried this. `asp-hide` doesn't prevent the partial from running - It only hides the content.  So the partial would fail if there was no alert as the  model would be null.